iOS/DuckDuckGo/NewTabPageViewController.swift

```diff
@@ -126,15 +123,17 @@ final class NewTabPageViewController: UIHostingController<AnyView>, NewTabPage {
 
         presentNextDaxDialog()
 
-        pixelFiring.fire(.homeScreenShown, withAdditionalParameters: [:])
-        sendDailyDisplayPixel()
-
```

Comment:
Moved to the main view controller - attach Home Screen function.  I tested this and it seemed to be firing at the same time as viewDidAppear.  I checked opening a new tab from the switcher and swiping to and from new tabs.